CMakeLists.txt

```diff
@@ -53,27 +178,13 @@ function(boost_http_proto_setup_properties target)
 
     target_link_libraries(${target}
         PUBLIC
-            Boost::align
-            Boost::assert
-            Boost::buffers
-            Boost::config
-            Boost::container_hash
-            Boost::system
-            Boost::throw_exception
-            Boost::url
-            Boost::utility
-            Boost::winapi
-    )
+            Boost::http_proto_core)
+
     if (ZLIB_FOUND)
-        target_compile_definitions(${target} PUBLIC BOOST_HTTP_PROTO_HAS_ZLIB)
+        target_link_libraries(boost_http_proto PUBLIC Boost::http_proto_zlib)
```

Comment:
LOL no... you can't conditionally link based on whether or not ZLIB is available...